_data/reference.yaml

```diff
@@ -91,6 +91,25 @@ toc:
     path: /reference/javascript.html
   - title: Python
     path: /reference/python.html
+- title: Pulumi Crosswalk for AWS
```

Comment:
With the move to Hugo, we can now define the TOC for this section in each page's front matter.

First, all the markdown files need to move to `/content/reference/crosswalk-aws` and `index.md` needs to be renamed `_index.md`.

Then you can follow a pattern similar to the files inside the [`/content/reference/vs`](https://github.com/pulumi/docs/tree/1ffc212d434c5d5933962941b3209c003ae12325/content/reference/vs) directory.

Inside `_index.md` add front matter like:

```
---
title: "Pulumi Crosswalk for AWS"
menu:
  reference:
    identifier: crosswalk-aws
    weight: 8
---
```

Regarding the `weight`: You'll need to tweak the weight in other files for the sections below this one, if you want `Pulumi Crosswalk for AWS` to come before `Compared to Others` and later sections.

Then in `vpc.md`:

```
---
title: "AWS Virtual Private Cloud (VPC)"
menu:
  reference:
    parent: crosswalk-aws
    name: "Virtual Private Cloud (VPC)"
    weight: 1
---
```

- `parent` is the parent menu item, the identifier we used in `_index.md`.
- `name` is an alternate name to use (otherwise if this isn't specified the title of the page is used)
- `weight` to order the items

And in `elb.md`:

```
---
title: "AWS Elastic Load Balancing (ELB)"
menu:
  reference:
    parent: crosswalk-aws
    name: "Elastic Load Balancing (ELB)"
    weight: 2
---
```

etc.

Or you could choose not to specify a weight for these child pages, and just let Hugo order them alphabetically based on the `name` (or `title` if a `name` wasn't specified).